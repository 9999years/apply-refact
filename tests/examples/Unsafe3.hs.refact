[("tests/examples/Unsafe3.hs:1:12: Warning: Redundant $\nFound:\n  unsafePerformIO $ Multimap.newIO\nWhy not:\n  unsafePerformIO Multimap.newIO\n",[Replace {rtype = Expr, pos = SrcSpan {startLine = 1, startCol = 12, endLine = 1, endCol = 44}, subts = [("a",SrcSpan {startLine = 1, startCol = 12, endLine = 1, endCol = 27}),("b",SrcSpan {startLine = 1, startCol = 30, endLine = 1, endCol = 44})], orig = "a b"}])]
