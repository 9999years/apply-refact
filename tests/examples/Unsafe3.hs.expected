slaves v = unsafePerformIO Multimap.newIO where foo = 1
